src/pghead.cpp

```diff
@@ -62,6 +62,7 @@ bool PgHead::GenerateFromMEIHeader(pugi::xml_document &header)
             }
             Text *text = new Text();
             text->SetText(UTF8to16(titleNode.node().text().as_string()));
+            rend->SetLang(titleNode.node().attribute("xml:lang").as_string());
```

Comment:
You need to check that `@xml:lang` is there, don't you?

Reply:
Actually it works fine this way. But if it'd make you feel better, I can addy it. 8-)

Reply:
I am surprised it is not crashing if the attribute is not there.

Reply:
You are right, for the doc of pugixml: "In case the attribute handle is null, both functions [as_string() and value()] return empty strings - they never return null pointers."